components/mediaLibrary.html

```diff
@@ -178,7 +178,9 @@
 
                             <!--
                             TODO: Discuss merits of lazy loading vs. background image
-                            <img ng-if="media.data.mime.indexOf('image') > -1" stratus-src="{{media.data.url || media.data.file}}">-->
+                            <img ng-if="media.data.mime.indexOf('image') > -1" data-plugin="lazy"
```

Comment:
This is rolling back a component to an unsupported version and is also commented out.  This really doesn't make sense.

Reply:
Thats my bad.. i suppose to commit mediaUpload.html but somehow it did not go through. fixed and squashed...